### PR TITLE
chore: add remaining language translations where necessary

### PR DESCRIPTION
**Pull request by abanuelo**
### Description

Working to add translations where missing across different pages/components. I may have missed a few spots so if you know that is the case please let me know and happy to get them into this pull request.

### Checklist

- [ ] I have tested the changes locally by running `npm run test`
- [ ] I have added or updated relevant documentation
- [ ] I have autoformatted the code by running `npm run eslint`
- [ ] I have added test cases (if applicable)

### CODE REVIEW

File: client/components/FormInputComponents/PhoneNumber/PhoneNumber.tsx [hunk 1]
```diff
@@ -36,11 +40,11 @@ const PhoneNumber = ({ field, index }: FieldProps) => {
         type="tel"
         placeholder="(201) 555-0123"
         {...register(`answers.${index}.phone_number`, {
-          required: required && "This field is required",
+          required: required && t("required"),
           pattern: {
             value:
               /(\+?\d{1,3})?[-.\s]?(\(?\d{1,4}\)?)?[-.\s]?\d{1,4}[-.\s]?\d{1,4}[-.\s]?\d{1,9}/,
-            message: "Invalid phone number",
+            message: t("phoneNumber.invalid"),
```
Comment (elimanzo) [inline client/components/FormInputComponents/PhoneNumber/PhoneNumber.tsx:47]: Nice use of the error message handler + translation 🔥

File: client/components/LanguageDropdown/LanguageDropdown.tsx [hunk 2]
```diff
@@ -17,10 +20,18 @@ const languages: LanguageOption[] = [
 const LanguageDropdown: React.FC<LanguageDropdownProps> = ({
   handleSelect,
 }) => {
-  const { user } = useAuth0();
-  const currentUser = user;
+  const { getAccessTokenSilently } = useAuth0();
+  let currentUser: User;
```
Comment (elimanzo) [inline client/components/LanguageDropdown/LanguageDropdown.tsx:24]: Above we made `currentUser` to  a state variable and I was wondering if we can do the same here?

`const [currentUser, setCurrentUser] = useState<User | null>(null);`

File: client/components/LanguageDropdown/LanguageDropdown.tsx [hunk 3]
```diff
@@ -17,10 +20,18 @@ const languages: LanguageOption[] = [
...
   const [selectedLanguage, setSelectedLanguage] = useState<string>("en");
 
+  useEffect(() => {
+    (async () => {
+      const token = await getAccessTokenSilently();
+      const email = Cookies.get("email") || "";
+      currentUser = await fetchUser(email, token);
```
Comment (elimanzo) [inline client/components/LanguageDropdown/LanguageDropdown.tsx:31]: If we can make it to a state we can do the standard `setCurrentUser(...)`

**Review by elimanzo — APPROVED** [review]
Thanks for this PR and combing through all the needed translations! I just have small nits, and it's good to go off my end 💯

File: client/components/FormInputComponents/PhoneNumber/PhoneNumber.tsx (hunk 1 above)
Comment (abanuelo) [reply client/components/FormInputComponents/PhoneNumber/PhoneNumber.tsx:47]: [🙇](https://emojipedia.org/person-bowing)

File: client/components/LanguageDropdown/LanguageDropdown.tsx (hunk 2 above)
Comment (abanuelo) [reply client/components/LanguageDropdown/LanguageDropdown.tsx]: Done!

File: client/components/LanguageDropdown/LanguageDropdown.tsx (hunk 3 above)
Comment (abanuelo) [reply client/components/LanguageDropdown/LanguageDropdown.tsx]: Done!

**Pull request merged**